Cargo.toml

```diff
@@ -14,21 +14,23 @@ exclude = ["/.github", "/examples", "/scripts"]
 
 [dependencies]
 tokio = "1.0"
-rustls = { version = "=0.22.0-alpha.2", default-features = false }
+rustls = { version = "0.22.0-alpha.2", default-features = false }
 
 [features]
 default = ["logging", "tls12"]
-dangerous_configuration = ["rustls/dangerous_configuration"]
 early-data = []
 logging = ["rustls/logging"]
 secret_extraction = ["rustls/secret_extraction"]
 tls12 = ["rustls/tls12"]
 
 [dev-dependencies]
-argh = "0.1"
+argh = "0.1.1"
 tokio = { version = "1.0", features = ["full"] }
 futures-util = "0.3.1"
-lazy_static = "1"
+lazy_static = "1.1"
 webpki-roots = "=0.26.0-alpha.1"
 rustls-pemfile = "=2.0.0-alpha.1"
-webpki = { package = "rustls-webpki", version = "=0.102.0-alpha.2", features = ["alloc", "std"] }
+webpki = { package = "rustls-webpki", version = "=0.102.0-alpha.3", features = ["alloc", "std"] }
+
+[patch.crates-io]
+rustls = { git = 'https://github.com/rustls/rustls' }
```

Comment:
This is unused.

Reply:
Well, by tracking the latest changes from upstream rustls I mean I'm really using the version from main branch. So without the patch, that means we are going to use the crates.io version of rustls ([rustls - crates.io: Rust Package Registry](https://crates.io/crates/rustls/0.22.0-alpha.2)) which is not exactly the intent

Reply:
Ah, sorry, you're right. Anyway, I don't want to track a branch here without pinning a commit, and I'd prefer just publishing another rustls alpha.

Reply:
sure, I will remove the patch and pin the right version when the next rustls alpha is cooked on crates.io.